reference/cache.md

```diff
@@ -69,6 +68,16 @@ And the following routes are **not** cached:
 > **note**
 > Regular expressions in routes are **not** supported.
 
+## Cache behavior
+
+Cache is only applied to `GET` and `HEAD` requests.
+Responses with `Cache-Control` header set to `Private`, `No-Cache`,
+or `No-Store` are not cached. Also, this applies to responses with
+`Set-Cookie` header being set.
+
+Cache would also respect the `Vary` header in the response.
+Multiple versions of the cache will be created if `Vary` header is set.
```

Comment:
From http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cache_valid

> If the header includes the “Vary” field with the special value “*”, such a response will not be cached (1.7.7). If the header includes the “Vary” field with another value, such a response will be cached taking into account the corresponding request header fields (1.7.7). 

First we need a understanding :)